kernelci/db/kernelci_api.py

```diff
@@ -72,6 +72,11 @@ def unsubscribe(self, sub_id):
             self._filters.pop(sub_id)
         self._post(f'unsubscribe/{sub_id}')
 
+    def count_nodes(self, attributes: dict = None):
+        """Get the count of all nodes matching attributes"""
+        resp = self._get('count', params=attributes)
+        return resp.json()
+
```

Comment:
Maybe this should have been put after `get_nodes()` to keep the pub/sub methods grouped together, then the `nodes` ones together.  Not a big issue though.

Reply:
I wanted to maintain the alphabetical order for methods related to nodes. That's why I put it first after pubsub methods.

Reply:
But, `get_event()` is not related to nodes.  So it should at least be put after that.

Reply:
Oh, yes. I will update it.